src/if_statements.py

```diff
@@ -1058,6 +1059,22 @@ def build_switch_statement(
     """
     switch_body = Body(print_node_comment=context.options.debug)
 
+    # If there are any case labels to jump to the `end` node immediately after the
+    # switch block, emit them as `case ...: break;` at the start of the switch block
+    # instead. This avoids having "dangling" `case ...:` labels outside of the block.
+    remaining_labels = []
+    for index, case_label in context.case_nodes[end]:
+        if index == switch_index:
+            comment = f"switch {switch_index}" if switch_index != 0 else None
```

Comment:
this is the third time we repeat this logic, better make a helper out of it. (making SimpleStatement.comment into a list?)